lib/eclipse/Parser/Parser.cpp

```diff
@@ -337,12 +340,20 @@ void ParserState::handleRandomText(const string_view& keywordString ) const {
     std::stringstream msg;
     std::string trimmedCopy = keywordString.string();
 
+
     if (trimmedCopy == "/") {
         errorKey = ParseContext::PARSE_RANDOM_SLASH;
         msg << "Extra '/' detected at: "
             << this->current_path()
             << ":" << this->line();
-    } else {
+    }
+    else if (lastSizeType == OTHER_KEYWORD_IN_DECK) {
+      errorKey = ParseContext::PARSE_EXTRA_RECORDS;
+      msg << "Too many records in keyword: " 
+          << lastKeyWord
+          << ".\n";
+    }
```

Comment:
This is missing a call to `parseContext.handleError(ParseContext::PARSE_EXTRA_RECORDS, msg);`

Reply:
The call is in line 363, below the if/elses.